apps/dsp-app/src/app/system/projects/projects-list/erase-project-dialog/erase-project-dialog.component.html

```diff
@@ -0,0 +1,52 @@
+<app-dialog-header
+  [title]="'appLabels.projects.list.eraseConfirmationDialog.title' | translate"
+  [subtitle]="'appLabels.projects.list.eraseConfirmationDialog.subtitle' | translate">
+</app-dialog-header>
+
+<mat-dialog-content>
+  <div>{{ 'appLabels.projects.list.eraseConfirmationDialog.warning' | translate }}</div>
+  <mat-form-field>
```

Comment:
Don't you want to use app-common-input instead of all form fields  in order to stay consistent with the rest of the app ? (see reusable-project-form for an example)